.gitignore

```diff
@@ -53,3 +53,6 @@ auto/
 # Exclude temporary documentation build files
 docs/_build/
 !docs/technical_reference/run_*
+
+# exclude test file
+two_phase_batch_model/test_membrane_loop.py
```

Comment:
Let's not add such specific and limited-relevance files to `.gitignore`. Just take care to not add them when you make commits.

Reply:
Sounds good.